webapp/sources/api-doc/paths/healthcheck/all.yml

```
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+get:
+  summary: Get all checks status
+  description: Run and get the result of all checks
+  operationId: getHealthcheckResult
+  responses:
+    "200":
+      description: Healthcheck informations
```

```suggestion
      description: Healthcheck information
```

information is always singular